scripts/deploy-federated-nginx.sh

```diff
@@ -0,0 +1,99 @@
+set -o pipefail
+
+source "$(dirname "${BASH_SOURCE}")/util.sh"
+
+TEST_NS="test-namespace"
+
+# Prerequisite:
+# - a cluster scoped federation deployment with `cluster1` and
+#   `cluster2` joined as member clusters
+# - kubectl config use-context cluster1
+#
+echo "Starting deploying a ngnix application in federation cluster."
+
+echo
+echo "Creating a test namespace ${TEST_NS}."
+kubectl create ns ${TEST_NS}
+
+echo
+echo "Creating a ngnix deployment in namespace ${TEST_NS}."
+kubectl apply -n ${TEST_NS} -f ../example/sample1/configmap.yaml
+
+kubectl apply -n ${TEST_NS} -f ../example/sample1/deployment.yaml
+
+echo
+echo "Creating a ngnix service in namespace ${TEST_NS}."
+kubectl apply -n ${TEST_NS} -f ../example/sample1/service.yaml
+
+echo
+echo "Federating resources in namespace ${TEST_NS} to member clusters."
+kubefedctl federate ns ${TEST_NS} --contents --skip-api-resources 'pods,secrets,serviceaccount,replicaset'
+
+echo
+echo "Checking status of federated resources."
+# TODO: Verify
+
+echo
+echo "Querying status of deployment nginx applications from member clusters."
+IP1=$(minikube -p cluster1 ip)
+IP2=$(minikube -p cluster2 ip)
+
+util::wait-for-condition 'nodePort' "kubectl --namespace ${TEST_NS} --context cluster2 get service -o jsonpath='{.items[0].spec.ports[0].nodePort}' &> /dev/null" 30
+util::wait-for-condition 'nodePort' "kubectl --namespace ${TEST_NS} --context cluster1 get service -o jsonpath='{.items[0].spec.ports[0].nodePort}' &> /dev/null" 30
+PORT1=$(kubectl --namespace ${TEST_NS} --context cluster1 get service -o jsonpath="{.items[0].spec.ports[0].nodePort}")
+PORT2=$(kubectl --namespace ${TEST_NS} --context cluster2 get service -o jsonpath="{.items[0].spec.ports[0].nodePort}")
+
+util::wait-for-condition 'nginx web server ok in cluster2' "curl ${IP2}:${PORT2} &> /dev/null" 120
+util::wait-for-condition 'nginx web server ok in cluster1' "curl ${IP1}:${PORT1} &> /dev/null" 120
+curl ${IP1}:${PORT1}
+echo
+curl ${IP2}:${PORT2}
+echo
+
+echo
+echo "Changing index.html in federated configmap."
+kubectl patch federatedconfigmap web-file -n ${TEST_NS} --type=merge  -p '{"spec": {"template": {"data": {"index.html": "Hi K8s Fed!"}}}}'
+
+echo
+echo "Querying status of deployed nginx applications from member clusters."
+util::wait-for-condition 'web content changed in cluster2' "(curl ${IP2}:${PORT2} | grep '^Hi K8s') &> /dev/null" 120
+util::wait-for-condition 'web content changed in cluster1' "(curl ${IP1}:${PORT1} | grep '^Hi K8s') &> /dev/null" 120
+curl ${IP1}:${PORT1}
+echo
+curl ${IP2}:${PORT2}
+echo
+
+echo
+echo "Changing override of federated deployment nginx to cluster1 with 'replicas' to 2."
+kubectl --namespace ${TEST_NS} patch federateddeployment nginx --type=merge --patch \
+    '{"spec" : {"overrides": [{"clusterName" : "cluster1", "clusterOverrides": [{"path": "spec.replicas", "value" : 2}]}]}}'
+
+echo
+echo "Changing placement of federated deployment nginx to cluster2 with 'cluster' to 'cluster2'."
```

Comment:
-> Update placement to include only `cluster2` so that the deployment will be removed from `cluster1`.